### DISCUSSION TITLE
Bring back > **Note** and > **Warning**

### DISCUSSION THREAD

**Discussion by seanpm2001** [community/community#61484] [Repositories]
### Select Topic Area

Product Feedback

### Body

I noticed today while working in Markdown that 

> **Note** Note

> **Warning** Warning

No longer show the blue info icon, or the yellow caution symbol next to them. Why did this get removed? I loved this feature :(

When can we expect that it will be brought back?

**Comment by sinsukehlab** [comment]
Now highlighting syntax needs forced line breaks.
See: #16925

```
> **Note**\
> This is a note.
```

> **Note**\
> This is a note.

---

> [!NOTE]\
> Try new highlighting syntax.
> 
> ### Usage
> ```
> > [!NOTE]\
> > This is a note.
> ```

**Reply by seanpm2001** [reply]
So now I have to fix all of the files that use this? What happens when they decide to change it again? I might as well just not use it anymore.

**Reply by sinsukehlab** [reply]
Unfortunately, you have to fix them if you want highlights displayed. Please ask @dipree what has happened.
